metadata/workspace-names.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0"?>
+<?xml version="1.0" ?>
```

Comment:
I can't tell if this is intentional or not but it seems like a mistake or an unwarranted change.